src/test/functional/features/joint-hub-page.feature

```diff
@@ -10,6 +10,11 @@ Feature: Joint hub page
     And a superuser updates "issueDate" with "2020-01-01"
 
   Scenario: Joint hub applicant 1 and applicant 2 pages
+    Given I set the case state to "AwaitingFinalOrder"
+    And I go to "/"
+    Then the page should include "You can now apply for a ‘final order’."
+    And the page should include "Apply for final order"
```

Comment:
I'd put this later in the test just for chronological order, but it shouldn't matter either way